docs/source/en/perf_train_cpu_many_k8s.md

```diff
@@ -0,0 +1,276 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Efficient Training on Multiple CPUs from a Kubernetes Cluter
+
+Using multiple CPUs accelerates the model training process by distributing the workload across multiple nodes. This
+guide builds upon the [Distributed CPU Training](https://huggingface.co/docs/transformers/perf_train_cpu_many) example
+to move the workload to a [Kubernetes](https://kubernetes.io) cluster.
+
+[Kubeflow](https://www.kubeflow.org) simplifies the process of deploying machine learning workloads to Kubernetes
+clusters. For this example, the [Kubeflow PyTorchJob training operator](https://www.kubeflow.org/docs/components/training/pytorch/)
+is used to deploy the distributed training job to the cluster. For efficient training on CPUs,
+[Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch) and
+[Intel® oneCCL Bindings for PyTorch](https://github.com/intel/torch-ccl) libraries are used.
+
+## Cluster setup
+
+> If [role-based access control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) is enabled on the
+> cluster, listing nodes and many other cluster wide commands require specific roles to be granted to the user.
+
+<!--
+TODO: In the case where RBAC prevents a user from seeing the nodes, what alternative command can we provide to allow the
+      user to verify that their kubectl is properly configured. kubectl get pods? kubectl get ns?
+-->
```

Comment:
In that case, things could get complicated and basically available commands to the user are unknown.
I think the best way to get around this is to mention that:
Please consult cluster admin to get a list of nodes available to the user's group.